src/pages/komodo-defi-framework/api/v20/enable_bch_with_tokens/index.mdx

```diff
@@ -21,6 +21,9 @@ The Komodo DeFi Framework supports Bitcoin Cash SLP tokens. Using this method, y
 | requires\_notarization   | boolean          | Optional, defaults to `true`. Has no effect on BCH.                                                                                                                                                                                                       |
 | tx\_history              | boolean          | Optional, defaults to `true`. If `true` the Komodo DeFi Framework API will preload transaction history as a background process. Must be set to `true` to use the [my\_tx\_history](/komodo-defi-framework/api/legacy/my_tx_history/#my-tx-history) method |
 | utxo\_merge\_params      | object           | A standard [UtxoMergeParams](/komodo-defi-framework/api/common_structures/activation/#utxo-merge-params) object. Used to reduce a wallet's UTXO count in cases where it is causing significantly slower RPC responses.                                    |
+<DevComment>
```

Comment:
write this a note. devcomment is not rendered on the webpage

Reply:
Done.